includes/blocks/podcast/markup.php

```diff
@@ -46,8 +47,26 @@
 	$term_image_id = '';
 }
 
+/*
+ * If not on a single post, set isDocked to none.
+ *
+ * This is to prevent having multiple instances of the podcast block with same docked position.
+ */
+$is_docked = ! is_single() ? 'none' : $attributes['isDocked'];
```

Comment:
One thing I missed during earlier rounds of testing is that there is no docking on pages or CPTs as this uses `is_single()` rather than `is_singular()`.

```suggestion
$is_docked = ! is_singular() ? 'none' : $attributes['isDocked'];
```

If this is the intent, then it is good for merge but I wanted to check before hand.

Reply:
Thanks, @peterwilsoncc! I updated `is_single()` with `is_singular()`.